src/pages/Login/LoginPage.tsx

```diff
@@ -72,6 +85,25 @@ export class LoginPage extends React.Component<LoginProps, LoginState> {
     if (authenticationConfig.strategy === AuthStrategy.openshift) {
       // If we are using OpenShift OAuth, take the user back to the OpenShift OAuth login
       window.location.href = authenticationConfig.authorizationEndpoint!;
+    } else if (authenticationConfig.strategy === AuthStrategy.token) {
+      if (this.state.password.trim().length !== 0 && this.props.authenticate) {
+        this.props.authenticate('', this.state.password);
+        this.setState({
+          showHelperText: false,
+          errorInput: '',
+          isValidToken: true,
+          filledInputs: true
+        });
+      } else {
+        const message = 'Please, provide a Service Account token.';
```

Comment:
Is this comma necessary? (commas in english are weird)

Reply:
The comma should be removed, but not a blocker, only if there is another revision.